molo/core/styles/wagtail-admin/modules/_m-filter.scss

```diff
@@ -64,4 +68,10 @@ header {
       padding: 0 5px 0 5px;
     }
   }
+
+  body.admin-view  {
+      .changelist-filter {
+        position: relative;
```

Comment:
@Mitso The Fly out doesn't like the relative position
![screen shot 2017-11-23 at 12 11 07](https://user-images.githubusercontent.com/9653693/33167908-bc6281f4-d047-11e7-936c-ea7d9f261f00.png)